modules/registrars/openprovider/OpenProvider/API/ApiInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace OpenProvider\API;
+
+interface ApiInterface
+{
+    /**
+     * @param string $cmd
+     * @param array $args
+     * @return ResponseInterface
+     */
+    public function call(string $cmd, array $args = []): Response;
```

Comment:
For interface lets return ResponseInterface here, not Response, because Response is concrete class, but ResponseInterface is abstraction for Response.